cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionCheckCommand.java

```diff
@@ -496,10 +507,11 @@ private void checkForHardFail(List<CliCheckResult> results) {
   }
 
   private List<CliCheckResult> executeChecks(
-      CliCheckerExecutor cliCheckerExecutor, List<AssetExtractionDiff> assetExtractionDiffs) {
+      CliCheckerExecutor cliCheckerExecutor,
+      Map<String, List<AssetExtractionDiff>> assetExtractionDiffs) {
     consoleWriter.newLine().a("Running checks against new strings").println();
-    return cliCheckerExecutor.executeChecks(assetExtractionDiffs).stream()
-        .collect(Collectors.toList());
+    return new ArrayList<>(
+        cliCheckerExecutor.executeChecks(assetExtractionDiffs, ALL_CHECKER_TYPES_KEY));
```

Comment:
What's the purpose of the `ALL_CHECKER_TYPES_KEY` string?

Reply:
Updated, please review